common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -5,8 +5,10 @@
 'video/03_video_player.js',
 ['video/02_html5_video.js', 'video/02_html5_hls_video.js', 'video/00_resizer.js', 'hls', 'underscore', '../time.js'],
 function(HTML5Video, HTML5HLSVideo, Resizer, HLS, _, Time) {
+    var config = null;
     var dfd = $.Deferred(),
         VideoPlayer = function(state) {
+            config = state.config;
```

Comment:
why we need this? I think state already has `config`. In order handle `isComplete` check we should set `isComplete` value [here](https://github.com/edx/edx-platform/blob/1e4daf749cba3f469423bf08fd4c202d87fba17a/common/lib/xmodule/xmodule/js/src/video/09_completion.js#L45)

and you can also set `enableNextOnCompletion` there too. 
`self. enableNextOnCompletion = this.state.config.enableNextOnCompletion`
and use it in rest of module like this
`self. enableNextOnCompletion`